resources/build/build-utils-ci.inc.sh

```diff
@@ -7,6 +7,38 @@
 
 . "$KEYMAN_ROOT/resources/build/jq.inc.sh"
 
+#
+# Returns 0 if current build is running in CI, as a pull request test, or as a
+# mainline branch test, or as a release build
+#
+builder_is_ci_build() {
+  if builder_is_ci_release_build || builder_is_ci_test_build; then
```

Comment:
Do some build configs use a `--ci` flag?

Reply:
They do at present. We'll move away from that in the future, because it should be unnecessary. Similarly, web and iOS have a ci.sh which is just different to everything else (there's quite a lot of stuff in the web build scripts which has diverged from the other platforms -- we should be able to simplify). (web scripts also do other tests for CI environments which should be updated to use these functions in the future)

Those changes can wait until 19.0, however.

Reply:
Tracked in #12640